app/src/main/java/com/infomaniak/drive/utils/IGeniusScanUtils.kt

```diff
@@ -30,13 +30,13 @@ interface IGeniusScanUtils {
 
     fun Context.initGeniusScanSdk() = true
 
-    fun Context.startScanFlow(resultLauncher: ActivityResultLauncher<Intent>) {
+    fun Activity.startScanFlow(resultLauncher: ActivityResultLauncher<Intent>) {
         MaterialAlertDialogBuilder(this, R.style.DialogStyle)
             .setTitle(R.string.allErrorFeatureNotAvailableInFdroid)
             .setPositiveButton("Ok") { _: DialogInterface?, _: Int -> }
             .show()
     }
 
-    fun Fragment.scanResultProcessing(intent: Intent, folder: File?) = Unit
+    fun Activity.scanResultProcessing(intent: Intent, folder: File?) = Unit
```

Comment:
When do we put extensions in `Extensions.kt` file  and when we do not ?

Reply:
We put it in `Extension.kt` when the function is used in more than one file, but it hasn't its own Utils 
Creating a dedicated Utils for a functionality takes over all other files, so we regroup all functions linked to this functionality in the Utils, even if it's an extension

Reply:
This allows to keep related code in the same file, and also to reduce the `Extensions.kt size`, as it starts to get really big